src/client/layouts/MinimalLayout.tsx

```diff
@@ -27,19 +27,18 @@ interface MinimalLayoutProps {
 	pageHeader: string;
 	leadText?: React.ReactNode;
 	imageId?: DecorativeImageId;
-	centered?: boolean;
```

Comment:
Adding `centered` was a change made in https://github.com/guardian/gateway/pull/3177 to make the welcome page look a bit less odd without an illustration and now it is no longer required.